packages/jest-types/src/Config.ts

```diff
@@ -11,6 +11,8 @@ import type {ForegroundColor} from 'chalk';
 
 type CoverageProvider = 'babel' | 'v8';
 
+type Timers = 'real' | 'fake' | 'modern';
```

Comment:
If it allows `modern`, it makes sense for it to also allow `legacy`. We should add that to `Timers` (not just in `ProjectConfig`)

Reply:
You're one hundred percent correct, I'll fix it right now.